### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by an-lee** [ZuodaoTech/everyone-can-use-english#1134] [Announcements]
## What's Changed
* Improve chat by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1084
* Upgrade deps by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1095
* Refactor Chat by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1108
* Upgrade dep by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1116
* Feat: New agent from templates by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1117
* Feat: Mention agent in chat by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1118
* Improve importing resources by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1120
* Feat: remove recordings in bulk by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1128
* Upgrade deps by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1129
* Refactor whisper by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1131
* Fix voice chat by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1132
* Upgrade deps by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1133

**Full Changelog**: https://github.com/ZuodaoTech/everyone-can-use-english/compare/v0.5.2...v0.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ZuodaoTech/everyone-can-use-english/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by EmmanuelAgui** [comment]
大佬，是不是这个版本对于上传的资源有作了压缩？现在下载下来的音频片段音质变差了，体验比较不好，希望可以原音频质量导入和片段下载。谢谢！

**Reply by an-lee** [reply]
是压缩了。对语言类音频的音质也这么敏感吗？

**Reply by EmmanuelAgui** [reply]
竟收到大佬回复，谢谢！
嗯嗯听起来缺了质感，练习起来有点怪怪，似乎没有那么纯正的感觉了（可能是错觉，但从个人体验来还是觉得不好哈），就回去下载旧的版本重新下载音频片段，不过现在版本自动更新似乎没有开关选项，被动升级后得重新安装旧版本~~~~大佬可以权衡一下是否帮忙处理这个问题哈，谢谢~
另，借此表达感谢，非常感恩这个工具对我的英语学习有很大的帮助~

**Reply by an-lee** [reply]
可以考虑加个是否压缩的选项。

**Reply by EmmanuelAgui** [reply]
Thanks a lot!

**Reply by EmmanuelAgui** [reply]
> 可以考虑加个是否压缩的选项。

哈咯大佬，在7.2的是否压缩选项好像没有生效，选择保持源文件，但导入之后文件还是被压缩了哈，还请大佬帮忙看下哈，谢谢~

**Reply by an-lee** [reply]
确实是有问题。在 v0.7.3 版本中修复了。
